src/WorkerChannel.ts

```diff
@@ -8,7 +8,27 @@ import { IEventStream } from './GrpcService';
 import { toTypedData } from './Converters';
 import { systemError } from './utils/Logger';
 
-export class WorkerChannel {
+/**
+ * The worker channel should have a way to handle all incoming gRPC messages.
+ * This includes all incoming StreamingMessage types (exclude *Response types and RpcLog type)
+ */
+interface IWorkerChannel {

```

Comment:
Move this to its own file

Reply:
It's a project-wide pattern to have the interface in the same file, so i'm hesitant to change that for just one class. I think we should either change the pattern throughout the project or keep the pattern as is for consistency?